src/test/java/seedu/address/logic/commands/deliverable/ClearCommandTest.java

```diff
@@ -16,10 +18,10 @@
     @Test
     public void execute_emptyDeliverableBook_success() {
```

Comment:
Rename method to execute_emptyDeliverableBook_faill(), since it is a negative test case